### PR TITLE
update readme for better clarity

### PR DESCRIPTION
**Pull request by sabrina-li**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -313,9 +323,6 @@ Using `withOptions` will **not** modify the options initially provided, but retu
   const { events } = init({ apiKey: '<YOUR_API_KEY>' });
 
   const options: FSRequestOptions = { 
-    // integrationSource is generally intended for FullStory
-    // or FullStory partners while building integrations.
-    //Generally it should be left empty.
```
Comment (patrick-fs) [inline README.md:318]: This is a good doc string to include in the source code - so that it shows up in IDEs

File: README.md [hunk 2]
```diff
@@ -25,12 +25,22 @@ yarn add @fullstory/server-api-client
...
 import { init } from '@fullstory/server-api-client';
 
-const fsClient = init({ apiKey: '<YOUR_API_KEY>' });
+const fsOpts: FullStoryOptions = {
+  // apiKey is required to init the FullStory client
+  apiKey: '<YOUR_API_KEY>',
+  
+  // integrationSource is generally intended for FullStory developers
+  // or FullStory partners while building integrations.
+  //Generally it should be left empty.
```
Comment (patrick-fs) [inline README.md:39]: This is a good doc string to include in the source code too - so that it shows up in IDEs

Comment (patrick-fs) [reply README.md:39]: Can up update this to include guidance around FullStory developers and Partners: https://github.com/fullstorydev/fullstory-node-sdk/blob/9788d9cf9a0bbb7b11bb0835e32294fc827d9414/src/http/options.ts#L5

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sabrina-li) [reply README.md:318]: Hmm agree that vast majority of the clients will not use `withOptions`. I added some thought on use case here https://fullstory.atlassian.net/browse/ECO-8823 
Perhaps it should be dropped from documentation altogether and add a comment on the method for clarity?

[the remark this answers is not in the document]
Comment (patrick-fs) [reply README.md:318]: Yeah, I think removing the request options from the ReadMe will help. Why don't you start there. Whether/what changes need to be made to `withOptions`

File: README.md [hunk 3]
```diff
@@ -25,12 +25,22 @@ yarn add @fullstory/server-api-client
 
 #### Initializing the Client
 
-Use the `init` function to initialize the FullStory client with [your API key](https://developer.fullstory.com/server/v2/authentication/).
+Use the `init` function to initialize the FullStory client with [your API key](https://developer.fullstory.com/server/v2/authentication/). The `FullStoryOptions` used in `init` function will be used for all server API requests, unless overridden by [per request options](#request-options).
```
[the remark this answers is not in the document]
Comment (patrick-fs) [reply README.md:28]: Go ahead and remove the statement about per-request overrides given our comment thread elsewhere in this PR.

Comment (patrick-fs) [inline README.md:28]: Go ahead and remove the statement about per-request overrides given our comment thread elsewhere in this PR.

```suggestion
Use the `init` function to initialize the FullStory client with [your API key](https://developer.fullstory.com/server/v2/authentication/). The `FullStoryOptions` used in `init` function will be used for all server API requests.
```

File: README.md [hunk 4]
```diff
@@ -25,12 +25,22 @@ yarn add @fullstory/server-api-client
 
 #### Initializing the Client
 
-Use the `init` function to initialize the FullStory client with [your API key](https://developer.fullstory.com/server/v2/authentication/).
+Use the `init` function to initialize the FullStory client with [your API key](https://developer.fullstory.com/server/v2/authentication/). The `FullStoryOptions` used in `init` function will be used for all server API requests.
```
Comment (patrick-fs) [inline README.md:28]:
```suggestion
Use the `init` function to initialize the FullStory client with [your API key](https://developer.fullstory.com/server/v2/authentication/). The `FullStoryOptions` used in the `init` function will be used for all server API requests.
```

**Review by patrick-fs — APPROVED** [review]
LGTM. Thank you!

File: README.md [hunk 5]
```diff
@@ -354,13 +336,13 @@ Using `withOptions` will **not** modify the options initially provided, but retu
 
 ### Multiple batch import jobs
 
-  It is recommended to have one batch import job of a resource type at a given time. However in case you need to create multiple batch import jobs by calling `batchCreate` multiple times. The jobs may be concurrently executed. In this case that the server APIs may return rate limiting errors. It is recommended to adjust the `pollingInterval` option accordingly.
+  It is recommended to have one batch import job of a resource type at a given time. However in case you need to create multiple batch import jobs by calling `batchCreate` multiple times. The jobs may be concurrently executed. In this case that the server APIs may return rate limiting errors. It is recommended to adjust the `pollInterval` option accordingly.
```
Comment (sabrina-li) [inline README.md:339]: @patrick-fs this is for: https://fullstory.atlassian.net/browse/ECO-8825
it's actually a typo here, `pollInterval` is documented a line#323

Comment (patrick-fs) [reply README.md:338]: Cool, I see the options now.

Head's up that I reorganized a code sample a little bit with this commit: 83102ac8461bb0949eef02f4c61c641648408104

PTAL and feel free to drop it if you'd prefer not to merge the change

Comment (sabrina-li) [reply README.md:338]: Looks good. Thanks! Good to merge?

**Pull request merged**